zds/tutorialv2/utils.py

```diff
@@ -509,6 +510,7 @@ def get_content_from_json(json, sha, slug_last_draft, public=False, max_title_le
         fill_containers_from_json(json, versioned)
     else:
         # MINIMUM (!) fallback for version 1.0
+        # only support of ARTICLE and TUTORIAL
```

Comment:
rien d'autre que ce qui est dit :

on amis le support des manifest v1 (donc avant la zep-02) pour éviter 
les problèmes de compatibilité au moment du passage. Maintenant gustavi 
ne va pas rendre les tribunes compatibles v1.


Le 17/07/2016 à 01:30, victor felder a écrit :
>
> In zds/tutorialv2/utils.py 
> <https://github.com/zestedesavoir/zds-site/pull/3606#discussion_r71073424>:
>
> > @@ -509,6 +510,7 @@ def get_content_from_json(json, sha, slug_last_draft, public=False, max_title_le
> >          fill_containers_from_json(json, versioned)
> >      else:
> >          # MINIMUM (!) fallback for version 1.0
> > +        # only support of ARTICLE and TUTORIAL
>
> Que signifie ce commentaire ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/zestedesavoir/zds-site/pull/3606/files/efd78156f2162c15bf1337382a0c884a3a87e141#r71073424>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/ABc_xS9W1Kp3RVmTJyqNyrbQAFiiYJIGks5qWWmqgaJpZM4IdSmR>.
>